apps/calendar/style/modify_account_view.css

```diff
@@ -24,7 +24,7 @@
 }
 
 #modify-account-view.in-progress .during-progress > div {
-  display: inline-block;
+  display: block;
```

Comment:
Do I change the file again ? or giving a new pull request will be a better idea ?